.rhcicd/ci.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
```

Comment:
Can we try adding `yarn install --immutable` here? I think we just haven't installed `npm-run-all` yet.